packages/contracts/contracts/ChugSplashManager.sol

```diff
@@ -391,10 +393,12 @@ contract ChugSplashManager is OwnableUpgradeable, ReentrancyGuardUpgradeable {
     ) public view returns (address payable) {
         return (
             payable(
-                Create2.compute(
-                    address(this),
+                Create2.computeAddress(
```

Comment:
nit: We can use [this version](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/master/contracts/utils/Create2.sol#L44-L46) of `computeAddress`, which doesn't require a third argument